Jenkinsfile_CNP

```diff
@@ -52,26 +56,27 @@ env.IDAM_URL = "https://idam-api.aat.platform.hmcts.net"
 env.IDAM_S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
 env.BEFTA_S2S_CLIENT_ID = "am_org_role_mapping_service"
 env.OAUTH2_REDIRECT_URI = "http://am-role-assignment-service-aat.service.core-compute-aat.internal/oauth2redirect"
-env.OAUTH2_CLIENT_ID = "am_role_assignment"
 env.OAUTH2_ACCESS_TOKEN_TYPE = "OIDC"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY = "JUST_WARN"
 env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
-env.OAUTH2_SCOPE_VARIABLES = "openid%20profile%20roles%20authorities"
-env.IDAM_CLIENT_ID="am_role_assignment"
+env.OAUTH2_SCOPE_VARIABLES = "openid%20profile%20roles%20authorities%20create-user%20manage-user"
 env.OPENID_SCOPE_VARIABLES = "openid+profile+roles+authorities"
 env.MICROSERVICE_NAME = "am_org_role_mapping_service"
 env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal"
-env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
-
+env.CRD_URL = "http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal"
+env.USER_PROFILE_URL= "https://rd-user-profile-api-pr-575.service.core-compute-preview.internal"
 // Vars for Kubernetes PACT
-env.PACT_BROKER_FULL_URL = 'https://pact-broker.platform.hmcts.net'
+env.PACT_BROKER_FULL_URL = "https://pact-broker.platform.hmcts.net"
+env.AMQP_TOPIC_NAME= "rd-caseworker-topic-sandbox/subscriptions/BeftaFunctionalTest"
```

Comment:
I think, this would be required only for publishing message directly to topic otherwise it can be removed.

Reply:
We will need it in second part of the story, where we are publishing the message to topic.
PR: https://github.com/hmcts/am-org-role-mapping-service/pull/64/